src/main/java/com/google/maps/GeoApiContext.java

```diff
@@ -53,8 +54,15 @@
   private long errorTimeout = DEFAULT_BACKOFF_TIMEOUT_MILLIS;
 
   public GeoApiContext() {
+    this(null);
+  }
+
+  public GeoApiContext(Proxy proxy) {
     rateLimitExecutorService = new RateLimitExecutorService();
     client.setDispatcher(new Dispatcher(rateLimitExecutorService));
+    if(proxy != null) {
```

Comment:
If 'proxy == null', make sure we pass the appropriate call onto the okhttp client: setProxy(Proxy.NO_PROXY)